rpm/openssl-pkcs11-sign-provider.spec

```diff
@@ -0,0 +1,53 @@
+# Copyright (C) 2023 IBM Corp.
+# SPDX-License-Identifier: Apache-2.0
+
+Name:          openssl-pkcs11-sign-provider
+Version:       1.0.0
+Release:       %autorelease
+Summary:       A PKCS#11 provider for OpenSSL 3.0 (private key operations only)
+License:       Apache-2.0
+URL:           https://github.com/opencryptoki/%{name}
```

Comment:
URL -> Url

Reply:
Accordings to [1], `URL` is in capital letters. Maybe this is not case-sensitive at all...

[1] https://rpm-packaging-guide.github.io/#preamble-items